GoofyLightsEditor/FrameList.cpp

```diff
@@ -116,8 +117,7 @@ void FrameList::DeleteNode_Middle(int pos){
         // Delete the head node.
         head = current -> next;
         head -> prev = NULL;
-        // Delete Attached RGB structure here 
-        delete(current);
```

Comment:
This is the only issue I spot at the moment, you delete the RGB structure, but erased the line that deleted that node in the linked list. I'll have to look over this some more, but at this point it looks like the RGB structure has been freed, but not the node in the linked list that was connected to the deleted RGB structure.